experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusSerializer.test.ts

```diff
@@ -48,6 +48,10 @@ class TestMetricReader extends MetricReader {
   async onShutdown() {}
 }
 
+function ignoreComments(result: string): string {
+  return result.replace(/^#.*\n/gm, '');
```

Comment:
As the test is scrutinizing the string outputs, IMO this step unnecessarily increases the complexity of the tests.

Reply:
Should I add the help comments to the expected text in the assertions then?

Reply:
TBO, I don't think applying regexp to the string is a more straightforward approach compared to the current one. IMO, testing with bracket accessing without regexp can be easier to maintain.

Reply:
Okay, so go back to direct testing with `['method']()`, works for me.